src/gui/scene/Renderer.js

```diff
@@ -161,9 +280,12 @@ class Renderer {
         objectsToCheck.forEach(obj => {
             this.#raycaster.layers.mask = this.#raycaster.layers.mask | obj.layers.mask;
         });
-        let results = this.#raycaster.intersectObjects( objectsToCheck || this.#scene.children, true );
+        let results = this.#raycaster.intersectObjects( objectsToCheck || this.#globalScale.getNode().children, true );
         results.forEach(intersection => {
             intersection.rayDirection = this.#raycaster.ray.direction;
+            intersection.scenePoint = intersection.point.clone();
```

Comment:
important change to communicate to the team, if you ant to make sure your raycast works in milimeter use theses values (scenePoint and sceneDistance)

Reply:
Do these need changes as well then?
https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/blob/main/src/device/index.js#L355
https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/blob/main/src/device/index.js#L373
https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/blob/main/src/avatar/index.js#L461
https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/blob/main/src/avatar/index.js#L469
https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/blob/main/src/gui/spatialIndicator.js#L175
https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/blob/main/src/gui/ar/groundPlaneRenderer.js#L194

Reply:
Great, thanks!

Reply:
@Steve-KX-RL and @dangond-ptc  tagging you here for visibility – be aware of the new `scenePoint` and `sceneDistance` that Rik added to the raycasting results – you may need to access the point like `raycastIntersects[0].scenePoint` rather than `raycastIntersects[0].point` in future when working in millimeter scale.